packages/tokens/src/colors.ts

```diff
@@ -0,0 +1,4 @@
+export const colors = {
+  current: { value: 'currentColor' },
```

Comment:
Saw this before but not a fan tbh. Why not just use currentColor?

Reply:
I tracked this back to a PR I did [here](https://github.com/leather-wallet/extension/pull/4602/files) and it was already set as `current: { value: 'currentColor' }` then, I'm not sure why it's like that. 

I think we can remove it if `currentColor` will still work as that is used quite a lot in SVGs.